capture_gui/lib.py

```diff
@@ -194,7 +195,7 @@ def _browse(path):
             default_filename = "playblast"
 
         # Default to images rule (if rule exists in workspace)
-        images_rule = mc.workspace("images", q=1, fre=True)
+        images_rule = pm.workspace.fileRules["images"]
```

Comment:
Ah, thanks. Were you able to set up a reproducible scenario in which that error would come up? (I'm not able to reproduce it (testing under Maya 2016.)